Source/FuseJS/Tests/BundleTests.uno

```diff
@@ -0,0 +1,159 @@
+using Uno;
+using Uno.IO;
+using Uno.Collections;
+using Uno.Testing;
+using Uno.Threading;
+using Uno.Text;
+using FuseTest;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace FuseJS.Test
+{
+	/*
+		File path stuff cribbed from Fuse.FileSystem. That stuff should probably be made public/simpler for dogfooding.
```

Comment:
It's better to expand the PR than duplicate code. Duplicated code is pure evil. 😈